website/layouts/_default/contribute.html

```diff
@@ -60,8 +60,8 @@ <h2>Collaboration is Welcomed!</h2>
                             </div>
                         </a>
                     </div>
-                    <p>Join us for our weekly community meeting, held every Wednesday at <a href="https://www.timeanddate.com/worldclock/converter.html?iso=20210518T140000&p1=1440&p2=4826&p3=234&p4=195" title="Find the meeting time in your local timezone">10am Eastern</a>.</p>
-                    <p>The meeting <a href="https://github.com/enterprise-contract/community/issues?q=is%3Aopen+is%3Aissue+label%3Acommunity-meeting+" title="View upcoming meeting agenda">agenda and joining details are available as a Github issue</a> shortly before the meeting. </p>
+                    <p>Join us for our weekly community meeting, held every Wednesday at <a href="https://www.timeanddate.com/worldclock/converter.html?iso=20210518T133000&p1=1440&p2=4826&p3=234&p4=195" title="Find the meeting time in your local timezone">9:30am Eastern</a>.</p>
```

Comment:
Unrelated to this PR I guess, but the 20210518 in the timeanddate link seems weird. Also maybe it's not daylight saving friendly?